src/lib/web-component/log-view/log-view-element.js

```diff
@@ -64,6 +83,15 @@ export class LogView extends HTMLElement {
       rowsFragment.appendChild(logTableRowInstance);
     }
     this.tableBody.appendChild(rowsFragment);
+
+    this.updateFilteredRowsCount();
+
+    this.totalLogs += logs.length;
+    this.dispatchEvent(
+      new CustomEvent('logcountchange', {
+        detail: { totalLogs: this.totalLogs },
+      })
```

Comment:
Merge this logic with `updateFilteredRowsCount`.

Reply:
I renamed the method to `updateRowCountAndStyle`. Any suggestion?